### ISSUE TITLE
 Copay Spending password reset

### ISSUE THREAD

**Issue by bitdoggie** [bitpay/copay#7135]
Hi respectfully requesting help or guidance resting my spending password.  I had intended deleting my wallet and importing the backup and stills asking me for spending password to send my BTC. Also on settings it tells me it is possible to see my 12 word phrase on export wallet but there it ask me for my spending wallet again. So I'm without other options that I know of . i have the backup password for the backup of the wallet if it is of any help.
Thanks

**Comment by gabrielbazan7** [comment]
Are you sure you are using the correct wallet? Could you check if the switch for the spending password in settings is on ?

**Comment by bitdoggie** [comment]
Yes Gabriel is on. When I try to switch it off it ask me for the spending password...

**Comment by gabrielbazan7** [comment]
Try deleting the app data from your phone settings, and importing the wallet again.

**Comment by bitdoggie** [comment]
I also deleted from copay and imported on Bitpay

**Comment by gabrielbazan7** [comment]
With the 12 words ?

**Comment by bitdoggie** [comment]
No I do not have the 12 words just the text/file backup.

**Comment by gabrielbazan7** [comment]
Ok, old Copay versions exports the private key encrypted if you set the spending password. So that's why your spending password is always on. Let me ask if there is something you can do. But, I think the only way is that you remember your password.

**Comment by bitdoggie** [comment]
Ok Gabriel your help is much appreciated.

**Comment by parkermor** [comment]
@bitdoggie 
if you have text backup I can crack your password and recover your wallet if you want need help send me email : anywalletrecovery@gmail.com

**Comment by richardsuns** [comment]
@bitdoggie 
if you still have the text backup of the wallet I might be able to crack your password. we can discuss your problem in telegrm. my telegram id is: walletrecover
also you might need to have some information about your password structure.
